.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,70 @@
+name: Installation or build bug report
+description: Report installation or build bugs
+labels: ["Type: Bug"]
+body:
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Answers checklist.
+      description: Before submitting a new issue, please follow the checklist and try to find the answer.
+      options:
+        - label: I have read the component documentation [ESP-IDF Components](https://components.espressif.com) and the issue is not addressed there.
+          required: true
+        - label: I have updated my idf-component-manager branch (master or release) to the latest version and checked that the issue is present there.
+          required: true
+        - label: I have searched the issue tracker for a similar issue and not found a similar issue.
+          required: true
+  - type: dropdown
+    id: component
+    attributes:
+      label: What component are you using? If you choose Other, provide details in More Information.
+      multiple: false
+      options:
+        - bdc_motor
+        - cbor
+        - coap
+        - eigen
+        - esp_encrypted_img
+        - esp_jpeg
+        - esp_serial_slave_link
+        - expat
+        - fmt
+        - jsmn
+        - json_generator
+        - json_parser
+        - led_strip
+        - libsodium
+        - nghttp
+        - pcap
+        - pid_ctrl
+        - qrcode
+        - sh2lib
+        - test_app
+        - usb
+        - Other
```

Comment:
I think we should modify the PR template, so that when the new component is added, the developer is reminded that they should add the component to this list.

(Or possibly some pre-commit check that the components here, in upload-components.yml and in test_apps/CMakeLists.txt are the same?..)

Reply:
Created new issue #157 so we don't forget